xmake/plugins/pack/wix/main.lua

```diff
@@ -49,6 +50,406 @@ function _get_wix()
     return wix, oldenvs
 end
 
+-- get windeployqt tool for Qt applications
+function _get_windeployqt()
+    local windeployqt = find_tool("windeployqt")
+    if not windeployqt then
+        -- Try to find it in Qt installation
+        local qt = find_qt()
+        if qt and qt.bindir then
+            local windeployqt_path = path.join(qt.bindir, "windeployqt.exe")
+            if os.isfile(windeployqt_path) then
+                windeployqt = {program = windeployqt_path}
+            end
+        end
+        
+        if not windeployqt then
+            return nil
+        end
+    end
+    return windeployqt
+end
+
+-- detect if this is a Qt project
+function _is_qt_project(package)
```

Comment:
The functions `_is_qt_project` and `_get_windeployqt` are the same across different pack implementations (PR #6828 #6830 #6829). Perhaps it would be better to put them in the same place.